lib/commands/key/index.ts

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
I removed the ID from the output, since you have to provide the ID in the command params it's redundant and just clutters the output.

Old:
```
ID:   155617
NAME: GitHub Key 3

AAAAB3NzaC1yc2EAAAADAQABAAABAQCwUlR+9FZuQnctoUnhr/Gud+cR10vVg7GaB3C2uukezPFrAvYBBNwFX5E5ICcCFJh6q/75kThgsgSTh8y1cQIZqlStyh3vPvQ8VL1BcnJW/ZXecMrCPyup8CDnvqTNDJsVpm6yy1TgZCWDsHBA19uPnMn3gNoehDynAnZ8QVXDS+KuGMq3WN8e9J10OxLLnIJcM5y7TXdrNAKm+GODCAK2LEnD5MqQfY9Pzv1Zcmmlvp4y34XjoDK38z/C97NLW92MtCqAERPZZBnPypoBhWh2G3we/JTQwh3xgffMgF+VAxRFvnhdZLFvFudcxl0wxPG4uL1+yo4ph7SgIHyUxkv9
```

New (without Id)
```
 GitHub Key 3
 ────────────
AAAAB3NzaC1yc2EAAAADAQABAAABAQCwUlR+9FZuQnctoUnhr/Gud+cR10vVg7GaB3C2uukezPFrAvYBBNwFX5E5ICcCFJh6q/75kThgsgSTh8y1cQIZqlStyh3vPvQ8VL1BcnJW/ZXecMrCPyup8CDnvqTNDJsVpm6yy1TgZCWDsHBA19uPnMn3gNoehDynAnZ8QVXDS+KuGMq3WN8e9J10OxLLnIJcM5y7TXdrNAKm+GODCAK2LEnD5MqQfY9Pzv1Zcmmlvp4y34XjoDK38z/C97NLW92MtCqAERPZZBnPypoBhWh2G3we/JTQwh3xgffMgF+VAxRFvnhdZLFvFudcxl0wxPG4uL1+yo4ph7SgIHyUxkv9
```